<launch>
    <!--Arguments-->
    <arg name="num_clusters" doc="Number (int) of clusters to use in frontier segmentation" default="5"/>
    <arg name="expansion_radius" doc="Radius (int) in cells in which obstacles can be expanded" default="1" />

    <!--Main node-->
    <node name="frontier_finder" pkg="sair-p2-GeorgeKreye" type="find_frontiers.py" output="screen"
    args="$(arg num_clusters) $(arg expansion_radius)"/>
</launch>
